input/docs/cs/segmentation.md

```diff
@@ -0,0 +1,74 @@
+title: Segmentace
+---
+>Tento dokument pracuje s obrázkem: 
+
+* [Obrázek kytky](../media/kytka256.jpg) v proměnné `A` 
+
+# Vypočet prahové hodnoty
+``` matlab
+A = imread('kytka256.jpg');
+Gray = rgb2gray(A);
+prah = graythresh(Gray); % vypočítá globální prahovou hodnotu, metodou Otsu
+prah2 = multithresh(Gray,2); % vypočítá 2 prahové úrovně
+```
+# K-means
```

Comment:
kmeans je tady a potom i dole, proč na 2 místech? Chtělo by to sjednotit. Navíc kmeans je dost nevhodný nadpis, tady se nevytváří kmeans algoritmus, tady se spíš používá k segmentaci... Taky by bylo vhodné vysvětlit jakým způsobem a jak to souvisí s tím předchozím příkladem a s otsu metodou...